dsl-interpreter/src/types.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'node-yaml';
```

Comment:
yaml doesn't have a types package?

Reply:
not that i could find

Reply:
i just realized i should drop down one layer to js-yaml. will do that.